[docs] Global at rule is called font-face (#15865)
mui-org/material-ui: docs/src/pages/customization/typography/typography.md

```

Then, you can change the theme to use this new font.

> It requires use of the [`CssBaseline`](/components/css-baseline/) component to globally define Raleway as a font face.

Reviewer: ```suggestion
You need to use the [`CssBaseline`](/components/css-baseline/) component to globally define Raleway as a font face.
```
? cc @mbrookes

Author: Or maybe
```suggestion
In order to globally define Raleway as a font face [`CssBaseline`](/components/css-baseline/) component needs to be used.
```

Reviewer 2: Better.

> In order to globally define Raleway as a font face [`CssBaseline`](/components/css-baseline/) component needs to be used.

Reviewer 2: ```suggestion
In order to globally define Raleway as a font face the [`CssBaseline`](/components/css-baseline/) component needs to be used.
```
